osbuild/main_cli.py

```diff
@@ -14,15 +14,10 @@
 import osbuild
 import osbuild.meta
 import osbuild.monitor
+from osbuild.util.markup import vt
 from osbuild.objectstore import ObjectStore
```

Comment:
```suggestion
import osbuild.monitor
from osbuild.objectstore import ObjectStore
from osbuild.util.markup import vt
```

`util` is ordered after `objectstore` in the alphabet, isn't it?

It feels like I am the only one obsessing with alphabetical include-ordering. Why doesn't this bother everyone? How can you live with yourself knowing the includes are unordered?

Reply:
Haha, I'm used to autoformatting and I'm still internalizing the preferred import sorting here. Fixed.